src/components/search-results/vessels/vessels.component.js

```diff
@@ -8,10 +8,19 @@ import { VESSELS_PAGE } from "../../../root/root.constants.js";
 
 import "../search-results.css";
 
+import { getVesselViewPageWithFilter } from "../../../helpers/get-data"
+
 class FoundVessels extends Component {
   render() {
     const { vesselsList, total, searchWords } = this.props;
 
+    const firstVessel = vesselsList[0];
+
+    const vessel = {
+      ...firstVessel,
+      name: firstVessel._id,
+    };
```

Comment:
I chose to explicitly include the `_id` here as the name to make it consistent with the shared method `getVesselViewPageWithFilter`

The _id is always the `name` in this case
("search" function in the Realm app)
https://github.com/WildAid/o-fish-realm/pull/196/files#diff-0225a1f467df6cc9f67cbcb7c0f564859421d4c6043b373a062b7a9399cf9460R20

Reply:
Love it, great idea. Keep it consistent is almost always a good thing.